omc3/model/constants.py

```diff
@@ -29,4 +37,7 @@
 B2_ERRORS_TFS = "b2_errors.tfs"
 PLANE_TO_HV = dict(X="H", Y="V")
 
-ACCELERATOR_MODEL_REPOSITORY = Path("/afs/cern.ch/eng/acc-models/lhc")
\ No newline at end of file
+ACCELERATOR_MODEL_REPOSITORY = Path("/afs/cern.ch/eng/acc-models")
+MODIFIER_BRANCH = "operation/optics"
```

Comment:
Similarly, this could be `MODIFIERS_DIRECTORY` as it's not a branch

Reply:
well, it is a branch in the sens that a directory is a tree.

changed to `_SUBDIR_